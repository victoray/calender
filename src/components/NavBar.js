import React, {useState} from "react";
import {Header, Menu} from "semantic-ui-react";


const NavBar = ()=> {
    return (
        <Menu inverted className={"navbar"} borderless>
            <Menu.Item>
                <img src={"https://react.semantic-ui.com/logo.png"}  alt={""}/>
            </Menu.Item>
            <Menu.Item>
                <Header inverted>Calender<sup>TM</sup></Header>
            </Menu.Item>

        </Menu>
    )
};

export default NavBar;