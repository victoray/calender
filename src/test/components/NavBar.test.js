import React from "react";
import {shallow, mount} from "enzyme";
import NavBar from "../../components/NavBar";


it('should render the <NavBar/>', () => {
    const component = mount(<NavBar/>);
    expect(component).toMatchSnapshot();
});