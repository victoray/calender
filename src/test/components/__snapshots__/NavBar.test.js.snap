// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the <NavBar/> 1`] = `
<NavBar>
  <Menu
    borderless={true}
    className="navbar"
    inverted={true}
  >
    <div
      className="ui borderless inverted navbar menu"
    >
      <MenuItem>
        <div
          className="item"
          onClick={[Function]}
        >
          <img
            alt=""
            src="https://react.semantic-ui.com/logo.png"
          />
        </div>
      </MenuItem>
      <MenuItem>
        <div
          className="item"
          onClick={[Function]}
        >
          <Header
            inverted={true}
          >
            <div
              className="ui inverted header"
            >
              Calender
              <sup>
                TM
              </sup>
            </div>
          </Header>
        </div>
      </MenuItem>
    </div>
  </Menu>
</NavBar>
`;
